viral/basic_plots.py

```
@@ -0,0 +1,938 @@
+                    (df["stage"] == stage)
+                    & (df["rewarded"] == rewarded)
+                    & (df["genotype"] == "NLGF")
+                ]["n_responders"],
+            ).pvalue
+            print(p_value)
+            p_text = f"P = {p_value:.2g}"
+            axes[idx].text(i, text_y, p_text, ha="center", va="top")
+
+    plt.tight_layout(rect=[0, 0, 1, 0.95])
+    plt.savefig(
+        SERVER_PATH / "viral_plots" / "n_responders" / f"comparison_n_responders.png"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)